packages/base/src/msg/core.rs

```diff
@@ -107,6 +107,11 @@ pub enum QueryMsg {
     CurrentUnbondBatch {},
     #[returns(crate::state::core::UnbondBatch)]
     UnbondBatch { batch_id: Uint128 },
+    #[returns(crate::state::core::UnbondBatchesResponse)]
+    UnbondBatches {
+        limit: Option<usize>,
```

Comment:
please replace `usize` by `Uint64`

Reply:
Here, in `take` method, we should pass usize:
https://github.com/hadronlabs-org/drop-contracts/blob/4eae28df3698b0d91535d9491d25dcb3abeda7d2/contracts/core/src/contract.rs#L244
like here
https://github.com/hadronlabs-org/drop-contracts/blame/4eae28df3698b0d91535d9491d25dcb3abeda7d2/contracts/auto-withdrawer/src/contract.rs#L277